openmp/libomptarget/plugins-nextgen/amdgpu/src/rtl.cpp

```diff
@@ -183,6 +183,29 @@ Error asyncMemCopy(bool UseMultipleSdmaEngines, void *Dst, hsa_agent_t DstAgent,
 #endif
 }
 
+Expected<StringRef> getTargetTripleAndFeatures(hsa_agent_t Agent) {
```

Comment:
Don't think this can be a `StringRef`, as it's referencing the string in `llvm::SmallVector<char> ISAName(Length);` which goes out of scope.

Reply:
Right, I changed that to std::string, thanks for the catch!